ee/modules/110-istio/templates/podmonitor.yaml

```diff
@@ -35,7 +35,7 @@ spec:
       targetLabel: namespace
   selector:
     matchExpressions:
-    - key: istio.io/rev
+    - key: service.istio.io/canonical-name
```

Comment:
What happens if the label will be updated right now before updating the istio?
Do both istio versions set the `service.istio.io/canonical-name` label?